README.md

```diff
@@ -177,6 +178,24 @@ Restarting the SQL way:
 1. run `db-setup/nuke-db.sh`
 2. restart the spring app `gradle bootRun --args='--spring.profiles.active=dev'`
 
+### Rollbacks
```

Comment:
Both of these commands work locally for me, but they don't seem to respect `SR_DB_PORT` - I had to edit the JDBC connection string in `build.gradle` to manually add the port. Is this something that can be done?

Reply:
Good callout. How do the changes in c54af87 work for you? The plugin is now configured to read the port from the `SR_DB_PORT` environment variable if available and defaults to `5432`.

Reply:
Perfect - works like a charm!